### ISSUE TITLE
Full screen is not full screen in demo.

### ISSUE THREAD

**Issue by GeoffAtHome** [googlecast/cast-video-polymer#2]
The demo (http://googlecast.github.io/cast-video-polymer/demo.html) does not go full screen when full screen is selected. I think the following css needs to be added:

.overlay-video {
      width: 100%;
      min-width: 100%;
      position: relative;
      left: 0;
      top: 0;
    }

This works on chrome. I have not tested on other browsers. They may be a better solution to this bug.

**Comment by pengying** [comment]
Yeah you're right It's the demo that's broken not the element it self.  I'll update it soon.

**Comment by GeoffAtHome** [comment]
My html isn't great. I know what I had done 'fixed' the problem but was probably not a correct solution.
